templates/_base.html

```diff
@@ -111,70 +76,17 @@
     }
 
     function signOut() {
-        let auth2 = gapi.auth2.getAuthInstance();
-        csClient.signOut(auth2).then(responseJson => {
+        csClient.signOut().then(responseJson => {
             console.log('Signed out:');
             console.log(responseJson);
             window.location.reload();
         });
     }
 
-    function refreshSession() {
-        console.log('refreshing google sigin-in session');
-        let cu = auth2.currentUser.get();
-        cu.reloadAuthResponse().then(function(unused_auth_response) {
-            // Calling this sets the flask session cookie.
-            csClient.signIn(auth2.currentUser.get());
-        }).catch(() => {
-            console.error('Refreshing session failed.');
-            let ar = cu.getAuthResponse();
-            let expires_at = ar.expires_at;
-            if (expires_at < Number(new Date())) {
-                signOut();
-                console.log('Explicitly signing out because session expired.');
-            }
-        });
-
-    }
-
-    var googleUser = {};
-    function startApp() {
-        gapi.load('auth2', function(){
-          auth2 = gapi.auth2.init({
-              client_id: '{{google_sign_in_client_id}}',
-              cookiepolicy: 'single_host_origin',
-              scope: 'profile email'
-          });
-          if (document.getElementById('gSignInCustomBtn')) {
-            attachSignin(document.getElementById('gSignInCustomBtn'));
-          } else {
-            // Google Sign-In sessions only last 1-hour.  Refresh the
-            // session immediately after each page naviagation
-            // and once every 10 minutes the user sits on a page.
-            refreshSession();
-            window.setInterval(refreshSession, 10 * 60 * 1000);
-          }
-        });
-    }
-
-    function attachSignin(element) {
-      auth2.attachClickHandler(element, {},
-          function(googleUser) {
-            onSignIn(googleUser);
-          }, function(error) {
-              console.log(error);
-          });
-    }
-
-    function onLoad() {
-      startApp();
-    }
-
     function promptSignIn(e) {
```

Comment:
@jrobbins I am unclear about this function - How is this function being used? Now that we have a one-tap signin in a popup mode, do we still need this function?

Reply:
Let's do this in a follow-up PR

Reply:
I'll send a separate CL to change the signed-out star behavior.